gradle-plugin/build.gradle.kts

```diff
@@ -28,10 +28,11 @@ dependencies {
     // replace AGP dependency w/ gradle-api when we have source registering API available.
     compileOnly("com.android.tools.build:gradle:$agpBaseVersion")
     compileOnly(gradleApi())
-    implementation(project(":kotlin-analysis-api")) {
+    compileOnly(project(":kotlin-analysis-api")) {
         isTransitive = false
     }
-    implementation(project(":api"))
+    api(project(":api"))
```

Comment:
compileOnly

Reply:
KSPLogger is still needed at runtime classpath, actually `implementation` makes more sense here.